traits/account-set/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "account-set"
+version = "2.0.0-dev"
+authors = ["Joshy Orndorff"]
+repository = 'https://github.com/substrate-developer-hub/recipes'
+edition = "2018"
+license = "GPL-3.0-or-later"
```

Comment:
Is this really the license we want to use?

Reply:
Seems consistent with the [repo license](https://github.com/substrate-developer-hub/recipes/blob/master/LICENSE) yes?

Reply:
That's true but it doesn't really change my question.

Reply:
Ok, then I'm not sure I understand what the question is; can you elaborate? The `-or-later` thing is from the [spdx](https://spdx.org/licenses/) list I think if that's the question here.

Reply:
I am certainly not a license expert, but I always thought [GPL was kind of a "controversial" license to use](https://www.apache.org/licenses/GPL-compatibility.html). Substrate itself, as well as the Knowledge Base and Tutorials, are licensed under Apache; the node template is under the Unlicense. I guess I'm just curious _why_ this license was chosen for this repo.

Reply:
Substrate and polkadot are GPLv3, which is pretty much the standard for Parity projects (parity-ethereum and parity-btc is too). As for the exact motivation for that I bet others can elaborate more but afaik it's the license we prefer and often becomes the "default" people pick. Some of the building blocks we make are MIT/Apache2 to help sustain wider use.

I agree that sample code might benefit from a more permissive license but I bet that has been discussed elsewhere?
/cc @JoshOrndorff

Reply:
> Substrate and polkadot are GPLv3

Looks like [this changed recently](https://github.com/paritytech/substrate/pull/5947).

Reply:
I updated the license to GPL3 and documented my reasoning in https://github.com/substrate-developer-hub/recipes/pull/230.  I'm very open to changing the license as long as:
1. There is a good reason to do so.
2. We're confident it won't switch back and forth repeatedly.

I'm super happy to know that Substrate has begun the process of relaxing the license, and I consider that a fine reason to re-open the discussion. But in any case changing the repo license is beyond the scope of this PR. @danforbes you're more than welcome to open another PR against the `develop` branch that changes the license. :+1: